pkg/test/harness.go

```diff
@@ -415,6 +435,17 @@ func (h *Harness) Stop() {
 	}
 }
 
+// wraps Test.Fatal in order to clean up harness
+func (h *Harness) fatal(args ...interface{}) {
+	// clean up on fatal in setup
+	if !h.stopping {
+		// stopping prevents reentry into h.Stop
+		h.stopping = true
```

Comment:
I guess, `h.fatal()` is never meant to be called from a goroutine? Please add a comment to the method description